addons/modules/functions/fnc_moduleTransferOwnership.sqf

```diff
@@ -81,6 +91,16 @@ _clientNames = ["str_a3_om_common_definitions.incphone_44"] + _clientNames;
             _player
         };
     };
+
+    // set headless client script flags
+    if (_HCState < 2) then {
+        if (acex_headless) then {
```

Comment:
This won't be `false` if ACEX is not loaded. You probably want to check the config class.

Reply:
30ad1c6